spec/system/pages/home_spec.rb

```diff
@@ -30,9 +30,6 @@
 
         visit home_page_path
 
-        expect(page).to have_text "Featured"
-
-        expect(page).to have_link "National Guidelines", href: occupation_standards_path(national_standard_type: {guideline_standard: 1})
```

Comment:
Can we at least check that this "National Guidelines" text exists, even if we don't test for the link anymore?

Reply:
We are clicking on "National Guidelines" to get to the page where we check the link. Same is true for the other sections (State, Occupational Frameworks, etc)

Reply:
Oh you're right, sorry!